phpunit.xml.dist

```diff
@@ -10,6 +10,7 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <server name="KERNEL_DIR" value="app/" />
+        <env name="DATABASE_URL" value="sqlite://%kernel.project_dir%/var/data/blog_test.sqlite"/>
```

Comment:
Actually you are right but it still works :smile: 

Really weird what happens now when dumping the connection within a test to see the config:

```
  -_params: array:9 [
    "url" => "sqlite://%kernel.project_dir%/var/data/blog_test.sqlite"
    "host" => "%kernel.project_dir%"
    "port" => null
    "user" => "root"
    "password" => null
    "driver" => "pdo_sqlite"
    "driverOptions" => []
    "defaultTableOptions" => []
    "path" => "var/data/blog_test.sqlite"
  ]
```

This seems to be more correct:

```xml
        <env name="DATABASE_URL" value="sqlite:///var/data/blog_test.sqlite" />
```

==>

```
  -_params: array:9 [
    "url" => "sqlite:///var/data/blog_test.sqlite"
    "host" => "localhost"
    "port" => null
    "user" => "root"
    "password" => null
    "driver" => "pdo_sqlite"
    "driverOptions" => []
    "defaultTableOptions" => []
    "path" => "var/data/blog_test.sqlite"
  ]
```